### PR TITLE
Documentation fixes for python public API

### PR DESCRIPTION
**Pull request by makdharma**

### CODE REVIEW

File: src/python/grpcio/grpc/__init__.py [hunk 1]
```diff
@@ -900,88 +919,78 @@ def add_generic_rpc_handlers(self, generic_rpc_handlers):
...
     @abc.abstractmethod
     def stop(self, grace):
-        """Stops this Server's service of RPCs.
+        """Stops this Server.
 
-    All calls to this method immediately stop service of new RPCs. When existing
-    RPCs are aborted is controlled by the grace period parameter passed to this
-    method.
+    This method immediately stop service of new RPCs in all cases.
+    If grace period is specified, this method returns immediately
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/grpc/__init__.py]: "If a grace period is specified," or "If grace is not None,".

File: src/python/grpcio/grpc/__init__.py [hunk 2]
```diff
@@ -900,88 +919,78 @@ def add_generic_rpc_handlers(self, generic_rpc_handlers):
...
+    and all RPCs active at the end of the grace period are aborted.
 
-    This method may be called at any time and is idempotent. Passing a smaller
-    grace value than has been passed in a previous call will have the effect of
-    stopping the Server sooner. Passing a larger grace value than has been
-    passed in a previous call will not have the effect of stopping the server
-    later.
+    If grace period is not specified, then all existing RPCs are
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/grpc/__init__.py]: "If a grace period is not specified," or "If grace is None,".

File: src/python/grpcio/grpc/__init__.py [hunk 3]
```diff
@@ -900,88 +919,78 @@ def add_generic_rpc_handlers(self, generic_rpc_handlers):
...
-    passed in a previous call will not have the effect of stopping the server
-    later.
+    If grace period is not specified, then all existing RPCs are 
+    teriminated immediately and the this method blocks until the last
+    RPC handler terminates.
 
-    This method does not block for any significant length of time. If None is
-    passed as the grace value, existing RPCs are immediately aborted and this
-    method blocks until this Server is completely stopped.
+    This method is idempotent, i.e. can be called multiple times. Passing a smaller
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/grpc/__init__.py]: Drop the ", i.e. can be called multiple times.". If the reader doesn't know what idempotence is, this isn't the place to teach them.

Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/grpc/__init__.py]: You've lost "may be called at any time" from this text, the significance of which is that in contrast to `start`(), which can only be called on a stopped server, this method may be called on a stopped server, a started server, or a stopping server. This method is safe to call on a server that has never been started, and that's a bit different from mere idempotence.

File: src/python/grpcio/grpc/__init__.py [hunk 4]
```diff
@@ -1260,13 +1264,13 @@ def secure_channel(target, credentials, options=None):
     """Creates a secure Channel to a server.
 
   Args:
-    target: The target to which to connect.
+    target: The server address
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/grpc/__init__.py]: Missing period at the end of this line.

File: src/python/grpcio/grpc/__init__.py [hunk 5]
```diff
@@ -623,27 +638,32 @@ def __call__(self,
...
+    The typical use case is to debug or gain better visibility into gRPC
+    runtime's state.
 
     Args:
-      callback: A callable to be invoked and passed a ChannelConnectivity value
-        describing this Channel's connectivity. The callable will be invoked
-        immediately upon subscription and again for every change to this
-        Channel's connectivity thereafter until it is unsubscribed or this
+      callback: A callable to be invoked with ChannelConnectivity argument.
+        ChannlenConnectivity describes current state of the channel.
```
Comment (makdharma) [inline src/python/grpcio/grpc/__init__.py]: Done

File: src/python/grpcio/grpc/__init__.py [hunk 6]
```diff
@@ -890,7 +909,7 @@ def service_name(self):
 
 
 class Server(six.with_metaclass(abc.ABCMeta)):
-    """Services RPCs."""
+    """Services RPCs. """
```
Comment (makdharma) [inline src/python/grpcio/grpc/__init__.py]: Fixed

File: src/python/grpcio/grpc/__init__.py (hunk 1 above)
Comment (makdharma) [inline src/python/grpcio/grpc/__init__.py]: Done

File: src/python/grpcio/grpc/__init__.py (hunk 2 above)
Comment (makdharma) [inline src/python/grpcio/grpc/__init__.py]: Done

File: src/python/grpcio/grpc/__init__.py (hunk 3 above)
Comment (makdharma) [inline src/python/grpcio/grpc/__init__.py]: Done

Comment (makdharma) [inline src/python/grpcio/grpc/__init__.py]: added "maybe called at any time" back in.

File: src/python/grpcio/grpc/__init__.py (hunk 4 above)
Comment (makdharma) [inline src/python/grpcio/grpc/__init__.py]: Done

File: src/python/grpcio/grpc/__init__.py [hunk 7]
```diff
@@ -92,9 +93,10 @@ def running(self):
     This method does not block.
 
     Returns:
-      True if the computation is scheduled to take place in the future or is
-        taking place now, or False if the computation took place in the past or
-        was cancelled.
+        bool:
+        Returns True if the computation is scheduled for execution or currently
+        executing.
+        Returns Flase if the computation already executed or was cancelled.
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/grpc/__init__.py]: Typo "Flase".

File: src/python/grpcio/grpc/__init__.py [hunk 8]
```diff
@@ -900,88 +919,78 @@ def add_generic_rpc_handlers(self, generic_rpc_handlers):
...
-    passed in a previous call will not have the effect of stopping the server
-    later.
+    If a grace period is not specified, then all existing RPCs are
+    teriminated immediately and the this method blocks until the last
+    RPC handler terminates.
 
-    This method does not block for any significant length of time. If None is
-    passed as the grace value, existing RPCs are immediately aborted and this
-    method blocks until this Server is completely stopped.
+    This method is idempotent and maybe called at any time. Passing a smaller
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/grpc/__init__.py]: Typo "maybe" where what is meant is "may be".

**Pull request merged**